pom.xml

```diff
@@ -348,6 +348,7 @@
         <version.org.ow2.asm>6.0</version.org.ow2.asm>
         <version.org.picketbox.picketbox-commons>1.0.0.final</version.org.picketbox.picketbox-commons>
         <version.org.picketlink>2.5.5.SP12</version.org.picketlink>
+        <version.org.powermock>2.0.0-beta.5</version.org.powermock>
```

Comment:
@soul2zimate you just mentioned this but I am leaving this comment that we should use final versions anyway, as an explanation for adding the hold label. @mention me when we have a final version.

Reply:
This no longer has a pom change.